modin/distributed/dataframe/pandas/partitions.py

```diff
@@ -90,7 +90,7 @@ def unwrap_partitions(
             f"Only API Layer objects may be passed in here, got {type(api_layer_object)} instead."
         )
 
-    modin_frame = api_layer_object._query_compiler._modin_frame
+    modin_frame = api_layer_object._query_compiler._modin_frame  # type: ignore[attr-defined]
```

Comment:
Why is `# type: ignore[attr-defined]` needed?

Reply:
`mypy` requires that attribute `_modin_frame` be specified for `_query_compiler`.